### ISSUE TITLE
it doesn't work automatically and show nothing when running "git diff"

### ISSUE THREAD

**Issue by lzane** [kynan/nbstripout#69]
I follow the demo from youtube as follow.

```zsh
(one_shot_face_recognition_env)  zane@LZANELI-MB1  ~/Desktop/test  git init
Initialized empty Git repository in /Users/zane/Desktop/test/.git/
(one_shot_face_recognition_env)  zane@LZANELI-MB1  ~/Desktop/test   master  git add Untitled.ipynb
(one_shot_face_recognition_env)  zane@LZANELI-MB1  ~/Desktop/test   master ✚  git commit -a
[master (root-commit) 8bac162] qq
 1 file changed, 51 insertions(+)
 create mode 100644 Untitled.ipynb
(one_shot_face_recognition_env)  zane@LZANELI-MB1  ~/Desktop/test   master  nbstripout --install
(one_shot_face_recognition_env)  zane@LZANELI-MB1  ~/Desktop/test   master  nbstripout --status
nbstripout is installed in repository b'/Users/zane/Desktop/test'

Filter:
  clean = b'"/Users/zane/Desktop/one_shot_face_recognition_env/bin/python3" "/Users/zane/Desktop/one_shot_face_recognition_env/lib/python3.5/site-packages/nbstripout.py"'
  smudge = b'cat'
  required = b'true'
  diff= b'nbstripout -t'

Attributes:
  b'*.ipynb: filter: nbstripout'

Diff Attributes:
  b'*.ipynb: diff: ipynb'
(one_shot_face_recognition_env)  zane@LZANELI-MB1  ~/Desktop/test   master ●  git add Untitled.ipynb
(one_shot_face_recognition_env)  zane@LZANELI-MB1  ~/Desktop/test   master ✚  git diff --cached
```

But after running `git diff --cached` I can not see anything, it just empty. After running `nbstripout --uninstall`, `git diff --cached` work fine as usual.

It does not strip output until I run `nbstripout Untitled.ipynb` explicitly.

**Comment by marionzualo** [comment]
I had the same issue. I fixed by using version 0.3.0 instead of version 0.3.1.
```
conda install -c conda-forge nbstripout=0.3.0
```

**Comment by villasv** [comment]
Installing version 0.3.0 doesn't work everywhere because of the `-t` missing argument, though.

**Comment by christation** [comment]
@villasv I was able to reproduce this. 
Downgrading to 0.3.0, then installing in the repo and then upgrading to 0.3.1 worked for me.

**Comment by ar-jan** [comment]
I just ran into this. Instead of downgrading to 0.3.0, it seems that removing `*.ipynb diff=ipynb` from `.gitattributes` is enough.

**Comment by villasv** [comment]
@ar-jan but you'll probably lose the diff filter?

**Comment by ar-jan** [comment]
I'm not sure which setting does what exactly, but with just `*.ipynb filter=nbstripout` in my `.gitattributes` the diff did not include any outputs (`git diff --staged | grep outputs` gave `"outputs": []` only), whereas before with both filter and diff configured in the .gitattributes, I ended up with `outputs` contents being commited. (I believe the diff showed empty outputs, but I'm not certain.)

**Comment by kynan** [comment]
This is a bug in 0.3.1 which is fixed by #67. I'll cut a new release, apologies for the delay.

**Comment by kynan** [comment]
0.3.2 is now released!
